mysql-test/suite/galera/t/mdev-26175.test

```diff
@@ -0,0 +1,29 @@
+--source include/galera_cluster.inc
+
+#
+# MDEV-26175 : Assertion `! thd->in_sub_stmt' failed in bool trans_rollback_stmt(THD*)
+#
+--connection node_1
+SET sql_mode="no_zero_date";
+SET GLOBAL wsrep_max_ws_rows=1;
+CREATE TABLE t2 (a INT);
+CREATE TABLE t1 (a INT NOT NULL PRIMARY KEY) ENGINE=InnoDB;
+CREATE TRIGGER tgr BEFORE INSERT ON t1 FOR EACH ROW INSERT INTO t2 VALUES (0);
+CREATE TABLE ti (id BIGINT NOT NULL PRIMARY KEY) ENGINE=InnoDB;
```

Comment:
How is table `ti` relevant to this test? It is not involved in trigger processing and might confuse the reader.

Reply:
I normally do not change provided test case on MDEVs but in this case ti seems unnecessary.